critiquebrainz/db/review.py

```diff
@@ -179,7 +183,7 @@ def get_count(*, is_draft=False, is_hidden=False):
         return result.fetchone()[0]
 
 
-def update(review_id, *, drafted, text, license_id=None, language=None, is_draft=None):
+def update(review_id, *, drafted, text, rating, license_id=None, language=None, is_draft=None):
```

Comment:
IMHO, rating or text should explicitly declared to be None if a review is just text or just rating . That way, it will be easier to maintain integrity.
And, it seems I have missed to raise exception for IntegrityError at the start of update and create function. I will add that.